src/client/app/containers/app.ts

```diff
@@ -62,6 +63,10 @@ export class AppComponent {
             this.snippet = snippet;
         });
 
+        this._store.select(fromRoot.getSharing).subscribe(sharing => {
+            sharing ? this.isDisabled = true : this.isDisabled = false;
```

Comment:
@Zlatkovsky what do you think, should we create a seperate store for dialog so that we can just disable the menu whenever a dialog is visible or should we just use this sharing store because it covers all the dialog/menu disabling scenarios today.